subsys/suit/storage/src/suit_storage_mpi.c

```diff
@@ -85,7 +85,8 @@ suit_plat_err_t suit_storage_mpi_configuration_load(suit_manifest_role_t role, c
 			(const suit_manifest_class_id_t *)ex_mpi->class_id;
 
 		if (entries[i].role == role) {
-			LOG_ERR("Manifest with role 0x%x already configured at index %d", role, i);
+			LOG_ERR("Manifest with role 0x%x%s already configured at index %d", role,
```

Comment:
Perhaps following log would be more informational:

'Manifest class ID assigned for multiple roles (current: %s, previous: %s)'

LOG_ERR("\t" SUIT_MANIFEST_CLASS_ID_LOG_FORMAT,
				SUIT_MANIFEST_CLASS_ID_LOG_ARGS(class_id));

Reply:
Changed to log the current and previous values for both role and class ID conflicts.